main.go

```diff
@@ -269,13 +302,10 @@ func tsFieldToTimestamp(input interface{}) (*time.Time, error) {
 	return &zeroTime, fmt.Errorf("don't know how to turn %T (value %s) into a time.Time object", input, input)
 }
 
+// Using the log fields of stack driver: https://cloud.google.com/logging/docs/structured-logging
```

Comment:
This is sadly not totally true. StackDriver had its own format and was acquired by Google at some point. They are now using the new format for which you added the log line but this function was seems it's not fully dealing with old logging.

I prefer that we keep backward compatibility, tests should sufficient to ensure nothing is broken.